audio_video_recorder_server/RecordTask[] array
